docs/source/getting-started/installation.rst

```diff
@@ -325,6 +329,49 @@ To configure SCP access, you need access keys and the ID of the project your tas
   Multi-node clusters are currently not supported on SCP.
 
 
+
+VMware vSphere
+~~~~~~~~~~~~~~
+
+To configure VMware vSphere access, store the vSphere credentials in :code:`~/.vsphere/credential.yaml`:
+
+.. code-block:: shell
+
+    mkdir ~/.vsphere
+    touch ~/.vsphere/credential.yaml
+
+Here is an example of configuration within the credential file:
+
+.. code-block:: shell
+
+    vcenters:
+      - name: <your_vsphere_server_ip_01>
+        username: <your_vsphere_user_name>
+        password: <your_vsphere_user_passwd>
+        skip_verification: true # If your vcenter have valid certificate then change to 'false' here
+        clusters: [] # [] means all the clusters in the vSphere can be used by Skypilot
```

Comment:
To keep the instruction simple, we can combine the instruction for specifying the subset of clusters in the comment here.
```suggestion
        # Clusters that can be used by SkyPilot:
        #   [] means all the clusters in the vSphere can be used by Skypilot
        # Instead, you can specify the clusters in a list:
        # clusters:
        #   - name: <your_vsphere_cluster_name1>
        #   - name: <your_vsphere_cluster_name2>
        clusters: []
        
```